src/devices/machine/myb3k_kbd.cpp

```diff
@@ -182,7 +187,7 @@ INPUT_PORTS_START( myb3k_keyboard )
 	PORT_BIT( 0x0080U, IP_ACTIVE_HIGH, IPT_KEYBOARD ) PORT_CODE(KEYCODE_1)     PORT_NAME("1 !") PORT_CHAR('1')  PORT_CHAR('!') // 16
 
 	PORT_START("MYB3K_T4")
-	PORT_BIT( 0x0001U, IP_ACTIVE_HIGH, IPT_KEYBOARD ) PORT_CODE(KEYCODE_2) PORT_NAME("2 \"") PORT_CHAR('2')  PORT_CHAR('"') // 17
+        PORT_BIT( 0x0001U, IP_ACTIVE_HIGH, IPT_KEYBOARD ) PORT_CODE(KEYCODE_2) PORT_NAME("2 \"") PORT_CHAR('2')  PORT_CHAR('"') // 17
```

Comment:
This puts the indentation all over the place in the input ports.  One tab per indent level is the MAME standard.

Reply:
ok, i'll fix

Reply:
There is a tool fixing this if you compile it with TOOLS=1 and you then get a number of handy tools compiled from the MAME source. The one to use is called srcclean, I think it has a prepended 't', so 'tsrcclean'. Just use it on your files and they will be fine I think. I've just learnt this myself.

Reply:
Fixed